.ci/generate_test_report_lib.py

```diff
@@ -122,6 +122,16 @@ def plural(num_tests):
             ]
         )
 
+    if failures or return_code != 0:
+        report.extend(
+            [
+                "",
+                "If these failures are unrelated to your changes (e.g., tests are "
+                "broken or flaky at HEAD), please open an issue at "
+                "https://github.com/llvm/llvm-project/issues.",
```

Comment:
Is there any label ore assignee they should add, so that these can be found easily?

Reply:
Added a comment they should add the infrastructure label.